.github/workflows/stale-prs.yml

```diff
@@ -0,0 +1,97 @@
+name: Mark stale pull requests in Shepherd repo
+
+on:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    steps:      
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Check PRs and compile report
+        id: compile_report
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const fiveDaysAgo = new Date(new Date().setDate(new Date().getDate() - 5)); 
+            const threeDaysAgo = new Date(new Date().setDate(new Date().getDate() - 3));
+            let report = '';
+            let noReviewsReport = '';
+            let noChangesReport = '';
+            let needsReviews = '';
+            let needsChanges = '';
+            let needsMerge = '';
+
+            const prs = await github.paginate(github.rest.pulls.list, {
+              owner: 'rancher',
+              repo: 'shepherd',
+              state: 'open',
+              per_page: 100,
+            });
```

Comment:
Should these promises have a catch error part for the unhandled or rejected promises?

Reply:
Changed. Also included error handling for the report status